xbmc/utils/EGLUtils.h

```diff
@@ -83,6 +83,11 @@ class CEGLAttributesVec
     return m_attributes.data();
   }
 
+  ssize_t const Size() const
```

Comment:
```suggestion
  ssize_t Size() const
```

Never return plain types as `const`